Resources/input/fsh/referencedExamples.fsh

```diff
@@ -14,37 +14,37 @@ Usage: #example
 * active = true
 * name[0].use = #official
 * name[=].family = "Fürstin von Musterfrau"
-  * extension[0].url = "http://fhir.de/StructureDefinition/humanname-namenszusatz"
+  * extension[0].url = "https://fhir.de/StructureDefinition/humanname-namenszusatz"
   * extension[=].valueString = "Graf"
-  * extension[+].url = "http://hl7.org/fhir/StructureDefinition/humanname-own-name"
+  * extension[+].url = "https://hl7.org/fhir/R4/StructureDefinition/humanname-own-name"
```

Comment:
hier analog zu oben ("URL for this extension: http://hl7.org/fhir/R4/extension-humanname-own-name.html")

Reply:
... gilt für alle URL-Änderungen in diesem Dokument